test/CMakeLists.txt

```diff
@@ -68,8 +68,6 @@ tarantool_make_lua_path(LUATEST_LUA_PATH
     ${PROJECT_SOURCE_DIR}/?.lua
     ${PROJECT_SOURCE_DIR}/?/init.lua
     ${PROJECT_SOURCE_DIR}/test-run/lib/checks/?.lua
```

Comment:
The `checks` submodule is removed from test-run as well as `luatest`. I guess this path has no effect and can be removed.